pkg/webhooks/machine_webhook.go

```diff
@@ -1507,6 +1533,22 @@ func validateNutanixResourceIdentifier(resource string, identifier machinev1.Nut
 	return nil
 }
 
+func validateNutanixBootType(bootType machinev1.NutanixBootType) error {
+	parentPath := field.NewPath("providerSpec")
+	// verify the bootType configurations
+	// Type bootType field is optional, and valid values include: "", Legacy, UEFI, SecureBoot
+	switch bootType {
+	case "", machinev1.NutanixLegacyBoot, machinev1.NutanixUEFIBoot, machinev1.NutanixSecureBoot:
+		// valid bootType
+	default:
+		errMsg := fmt.Sprintf("valid bootType: \"\", %q, %q, %q.",
```

Comment:
this might be slightly confusing for a user, could we change this to say something like "valid bootType values are: ...`

Reply:
Changed.